.gitignore

```diff
@@ -8,3 +8,5 @@ node_modules
 key.json
 *.golden.png
 lerna-debug.log
+build/**
```

Comment:
Shouldn't just `build` and `dist` be enough here?

Reply:
yup you're right. I added that when they were already built. so I needed the `**`, but if you delete the directories and just have `build` and `dist` it works fine.